azurerm/resource_arm_automation_bool_variable.go

```diff
@@ -0,0 +1,42 @@
+package azurerm
+
+import (
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceArmAutomationBoolVariable() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmAutomationBoolVariableCreateUpdate,
+		Read:   resourceArmAutomationBoolVariableRead,
+		Update: resourceArmAutomationBoolVariableCreateUpdate,
+		Delete: resourceArmAutomationBoolVariableDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: AutomationVariableCommonSchemaFrom(map[string]*schema.Schema{
+			"resource_group_name": resourceGroupNameSchema(),
```

Comment:
Why isn't this part of the CommonSchema function?

Reply:
Fixed